std/algorithm/sorting.d

```diff
@@ -269,21 +301,85 @@ if (is(typeof(ordered!less(values))))
     return true;
 }
 
+deprecated("Please use isOrdered")
+bool isSorted(alias less = "a < b", Range)(Range r)
```

Comment:
By the way it's only possible because no partial speciliazation was needed. In the opposite case there's a _bug_ that would have break code: UFCS doesn't work anymore with partial speciliazation.